test/pdfs/issue14203.pdf.link

```diff
@@ -0,0 +1 @@
+https://github.com/mozilla/pdf.js/files/7428963/pdfcomment.pdf
```

Comment:
Nit: Missing a line-break at the end of the line.

Reply:
InsertFinalNewline of my editor was turned off,I would correct it and push again